cppcache/integration-test/testThinClientRemoteQuerySS.cpp

```diff
@@ -226,7 +226,7 @@ void compareMaps(HashMapOfCacheable &map, HashMapOfCacheable &expectedMap) {
 }
 
 void stepOne() {
-  initGridClient(true);
+  initClient(true);
```

Comment:
This may be a redundant test if it was initializing grid client. Is there another test that covers these same methods but didn't init grid?